src/command.c

```diff
@@ -125,6 +125,10 @@ const command_s cmd_list[] = {
 	{NULL, NULL, NULL},
 };
 
+#ifdef PLATFORM_HAS_CUSTOM_COMMANDS
+extern const command_s platform_cmd_list[];
```

Comment:
Given how this is defined, and as it decays to a pointer immediately, we're going to suggest defining this here as `extern const command_s *platform_cmd_list;` as that's a little more truthful to what's actually going to go on here.

Reply:
Fixed